### PR TITLE
Add authentication for Custom RTMP servers

### PR DESCRIPTION
**Pull request by chaironeko**
Fixed authentication (USE_ONLY_MD5) in rtmp.c
Added UI in custom RTMP output plugin
I also added translations for the new UI strings and cross-checked they appear on website logins in the appropriate countries.

### CODE REVIEW

**Comment by ONSBalder** [comment]
Can one of the admins verify this patch?

File: plugins/rtmp-services/data/locale/zh-CN.ini [hunk 1]
```diff
@@ -3,4 +3,5 @@ CustomStreamingServer="自定义流媒体服务器"
 Service="服务"
 Server="服务器"
 StreamKey="流密钥"
-
+Username="用戶名"
+Password="密碼"
```
Comment (jiangtiandao) [inline plugins/rtmp-services/data/locale/zh-CN.ini]: New line copied from zh-TW. 
should be:
Username="用户名"
Password="密码"

**Comment by chaironeko** [comment]
Have made recommended changes to the commits. how does it look now? :)

**Comment by jiangtiandao** [comment]
OK now ^_^ @chaironeko

**Comment by dodgepong** [comment]
I should have been more explicit in the chat: all localization should be removed from this pull request except for en-US. The rest will be overwritten by Crowdin.

**Comment by chaironeko** [comment]
ok, all localisation removed. :)

File: plugins/obs-outputs/librtmp/rtmp.c [hunk 2]
```diff
@@ -4217,8 +4223,14 @@ RTMP_Close(RTMP *r)
 #if defined(CRYPTO) || defined(USE_ONLY_MD5)
     if (!(r->Link.protocol & RTMP_FEATURE_WRITE) || (r->Link.pFlags & RTMP_PUB_CLEAN))
     {
-        free(r->Link.playpath0.av_val);
-        r->Link.playpath0.av_val = NULL;
+        for (int idx = 0; idx < r->Link.nStreams; idx++)
+		{
```
Comment (palana) [inline plugins/obs-outputs/librtmp/rtmp.c]: Spotted a few tabs that should probably be spaces

**Comment by chaironeko** [comment]
Fixed the tabs in rtmp.c :)

**Pull request closed without merging**